### PR TITLE
added home page cards, and edited footer

### PR DESCRIPTION
**Pull request by Drewbrad4**
Added our home page links and a scroll function and a link to the footer.

### CODE REVIEW

File: app/javascript/components/Pages/BorrowShow.js [hunk 1]
```diff
@@ -24,14 +24,14 @@ const BorrowShow = ({ book }) => (
 
 export default BorrowShow;
 
-BorrowShow.propTypes = {
```
Comment (talk2MeGooseman) [inline app/javascript/components/Pages/BorrowShow.js:27]: NICE!

File: app/javascript/components/Pages/BorrowedShow.js [hunk 2]
```diff
@@ -25,14 +25,14 @@ const BorrowedShow = ({ book }) => (
 
 export default BorrowedShow;
 
-BorrowedShow.propTypes = {
```
Comment (talk2MeGooseman) [inline app/javascript/components/Pages/BorrowedShow.js:28]: NICE!

File: app/javascript/components/Pages/Home.js [hunk 3]
```diff
@@ -1,10 +1,53 @@
...
 
 export default class Home extends Component {
-    render(){
-        return(
+    render() {
+        return (
             <>
+
                 <h2>Home</h2>
+                <br></br>
```
Comment (talk2MeGooseman) [inline app/javascript/components/Pages/Home.js:11]: Instead of br tags, you can also try creating css styles to do padding on the bottom.

**Review by talk2MeGooseman — CHANGES_REQUESTED** [review]

**Comment by devJoelMichael** [comment]
Looks good

**Review by devJoelMichael — APPROVED** [review]

**Pull request merged**